hsds/attr_sn.py

```diff
@@ -764,3 +1008,217 @@ async def PUT_AttributeValue(request):
     resp = await jsonResponse(request, req_rsp)
     log.response(request, resp=resp)
     return resp
+
+
+async def _get_attributes(app, obj_id, attr_names,
+                          IncludeData=False,
```

Comment:
`IncludeData -> includeData`

Reply:
changed the calling code (wanted to keep the named argument same as the http query param